src/BSeries.jl

```diff
@@ -1775,22 +1778,32 @@ function energy_preserving_order(rk::RungeKuttaMethod, max_order)
 end
 
 """
-    is_energy_preserving(rk::RungeKuttaMethod, order)::Bool
-
```

Comment:
Please do not delete this part of the docstring - but adapt it to the new calling signature

Reply:
Yes, sorry.